lib/api/v1/report-controller.js

```diff
@@ -151,10 +162,7 @@ module.exports = function(app, user) {
         // Mark each report as flagged to catch it in model
         report.toggleFlagged(req.body.flagged);
         report.save(function(err) {
-          if (err) {
-            if (!res.headersSent) res.send(err.status, err.message)
-            return;
-          }
+          if (err) return res.send(err.status, err.message);
```

Comment:
Are these all reverted because it's handled automatically now in https://github.com/TID-Lab/aggie/pull/398 or is that separate?

Reply:
Separate. They were accidentally reverted, but I just reverted them back.